crates/turborepo-cache/src/http.rs

```diff
@@ -120,6 +126,20 @@ impl HTTPCache {
         }
     }
 
+    async fn log_fetch(&self, event: analytics::CacheEvent, hash: &str, duration: u64) {
+        // If analytics fails to record, it's not worth failing the cache
+        if let Some(analytics_recorder) = &self.analytics_recorder {
+            let analytics_event = AnalyticsEvent {
+                session_id: None,
+                source: analytics::CacheSource::Remote,
+                event,
+                hash: hash.to_string(),
+                duration,
+            };
+            let _ = analytics_recorder.send(analytics_event).await;
```

Comment:
Same comment as in the fs cache with respect to unbounded channel/or `blocking_send`